rules/native_binary.bzl

```diff
@@ -48,79 +47,39 @@ def _impl_rule(ctx, is_windows):
         runfiles = runfiles,
     )
 
-def _impl(ctx):
-    return _impl_rule(ctx, ctx.attr.is_windows)
-
 _ATTRS = {
     "src": attr.label(
         executable = True,
         allow_single_file = True,
         mandatory = True,
         cfg = "host",
+        doc = "path of the pre-built executable",
     ),
     "data": attr.label_list(allow_files = True),
```

Comment:
Nit: `data` attr needs a docstring since the macros and the macros' docs are gone

Reply:
Done